tests/test_chefsavvy.py

```diff
@@ -0,0 +1,59 @@
+# mypy: allow-untyped-defs
+
+from recipe_scrapers.chefsavvy import ChefSavvy
+from tests import ScraperTest
+
+
+class TestChefSavvyScraper(ScraperTest):
+
+    scraper_class = ChefSavvy
+
+    def test_host(self):
+        self.assertEqual("chefsavvy.com", self.harvester_class.host())
+
+    def test_author(self):
+        self.assertEqual("Kelley Simmons", self.harvester_class.author())
+
+    def test_title(self):
+        self.assertEqual("Slow Cooker Broccoli Beef", self.harvester_class.title())
+
+    def test_category(self):
+        self.assertEqual("Main Course", self.harvester_class.category())
+
+    def test_total_time(self):
+        self.assertEqual(310, self.harvester_class.total_time())
+
+    def test_yields(self):
+        expected_yields = "4 servings"
+        self.assertEqual(expected_yields, self.harvester_class.yields())
+
+
+def test_instructions(self):
+    expected_instructions = """Add steak, broth, soy sauce, garlic, oyster sauce, brown sugar, and sesame oil to a slow cooker. Stir to combine.
+Cook on low for 4-5 hours or until steak is tender.
+Reserve 1/4 cup of the cooking liquid and whisk in the cornstarch to the reserved liquid.
+Slowly stir the cornstarch mixture into the slow cooker along with the broccoli and continue cooking on low for an additional 30 minutes or until sauce has thickened and broccoli is tender.
+Serve immediately with rice and enjoy!"""
+
+    # Normalize the line endings and remove leading/trailing whitespaces
+    actual_instructions = (
+        self.harvester_class.instructions().strip().replace("\r\n", "\n")
+    )
```

Comment:
Hi @jknndy - thanks for your contribution!

Could you add one level of indentation to these lines (the `test_instructions` method)?  At the moment I think it's declaring a module-level method that contains local methods (`def test_ratings`, for example, is inside the scope of `test_instructions`, instead of in the scope of `TestChefSavvyScraper`).